### PR TITLE
Create quality-estimation.md

### PR DESCRIPTION
**Pull request by ovesyan**

### CODE REVIEW

**Comment by bittlingmayer** [comment]
> # Quality Estimation
> (What it is and motivation)
> 
> > English: ... 
> > (Just a concrete example of sentence-level input and output, one with a good translation and one with a bad translation.)
> 
> Sentence-level scores can be aggregated into paragraph-level scores or document-level scores.
>
> ## Use cases
> Quality estimation has offline and production use cases.
> - Evaluation of machine translation systems or models
> - [Hybrid translation]
> - Estimating human post-editing effort
> - Validating final human translations
> - Filtering training data for machine translation
>
> ## Approaches
> Supervised quality estimation trains on parallel data that includes human labels or human post-edits.  Unsupervised quality estimation trains on monolingual data or parallel data only.
>
> Blackbox quality estimation does not use ...  Glassbox quality estimation uses...
>
> Early quality estimation approaches use machine learning with feature engineering.  With the rise of deep learning, deep learning approaches 
>
> Multilingual quality estimation trains one model or system for many language pairs.
>
> 
> ## Systems
> ||||
> |-------------|--------------------|------------------------------|
> | QuEst++      | Framework             | University of Sheffield            |
> | DeepQuest  | Framework             | University of Sheffield            |
> | OpenKiwi     | Framework             | Unbabel                                  |
> | TransQuest  | Pretrained models | University of Wolverhampton |
> | KantanQES  | Feature of machine translation API | KantanAI       |
> | ModelFront  | API                         | ModelFront                              |
> 
> ## Research
> In 2012, Lucia Specia and Radu Soricut organized the first at Shared Task on Quality Estimation.
> In 2018, Lucia Specia, Carolina Scarton and Gustavo Enqrique Paetzold published the book *Quality Estimation for Machine Translation*.

> There is also research on word-level quality estimation and paragraph-level quality estimation.

> In 2020, ModelFront launched a multilingual quality estimation API and Facebook Research launched unsupervised quality estimation internally.
